packages/create-app/templates/template-tutorial-todo-app/src/TaskList.tsx.hbs

```diff
@@ -1,10 +1,10 @@
-import { OsdkTodoProject } from "{{osdkPackage}}/ontology/objects";
```

Comment:
For all of these files I think we can remove the `.hbs` extension as they no longer need templating (it doesn't make a big difference but maybe makes it clearer that there isn't anything special about these files)